test/client_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "./test_helper"
+require_relative "../src/clients/kdump"
+ 
+describe Yast::KdumpClient do
+ 
+  describe ".cmdKdumpStartup" do
+    let(:alloc_mem_high) {"100"}
+    let(:alloc_mem_low) {"50"}
+    
+    before(:each) do
+      subject.cmdKdumpStartup(options)
+    end
+   
+    context "When using alloc memory high and low" do
+      let(:options) {{"enable"=>"", "alloc_mem"=>"#{alloc_mem_low},#{alloc_mem_high}"}}
+    
+      it "sets the low and high values of  Kdump allocated_memory " do
+        expect(Yast::Kdump.allocated_memory).to eq({:low=>alloc_mem_low, 
```

Comment:
it looks like rubocop is not enabled because it would complain about the using `eq({:low=>alloc_mem_low...)` instead of just `eq(low: alloc_mem_low...)` :) But let's enable it in the future (not before a deadline like today).